frontend/pages/[resourceType].tsx

```diff
@@ -0,0 +1,111 @@
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next';
+import { Button, Grid, Divider } from '@mantine/core';
+import Link from 'next/link';
+import BackButton from '../components/BackButton';
+
+/**
+ * Component which displays list of all resources of some type as passed in by (serverside) props
+ * @returns component with list of resource (by id) buttons that are links to that resource's details
+ */
+export default function ResourceList({ ids, resourceType }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  const idItems = ids.map(id => {
+    return (
+      <Link href={`/${resourceType}/${id}`} key={id}>
+        <Button
+          fullWidth
+          color="cyan"
+          radius="md"
+          size="md"
+          variant="subtle"
+          styles={() => ({
+            root: {
+              padding: '2px'
+            },
+            inner: {
+              justifyContent: 'left'
+            }
+          })}
+        >
+          <div>
+            {resourceType}/{id}
+          </div>
+        </Button>
+      </Link>
+    );
+  });
+  return (
+    <div
+      style={{
+        width: '78vw'
+      }}
+    >
+      <div>
+        <BackButton />
+      </div>
+      <Grid columns={7}>
+        <Grid.Col offset={3} span={2} style={{ paddingTop: '6px' }}>
+          <h2 style={{ color: 'gray', marginTop: '0px', marginBottom: '8px' }}>{`${resourceType} IDs`}</h2>
+        </Grid.Col>
+        <Grid.Col
+          span={2}
+          style={{
+            paddingTop: '6px'
+          }}
+        ></Grid.Col>
```

Comment:
Now that we have the back button, the dividers (for resource counts and the resourceType page) are no longer aligned. I think we might want to put the back button component in a `Grid.Col` so that it is aligned with the header without affecting the location of the divider.

Played around with this a little, and this achieved what I was looking for (not sure if it's the most efficient... I'm not the best with grids):

```javascript
<Grid columns={7}>
  <Grid.Col offset={0} span={1}>
    <div >
      <BackButton />
    </div>
  </Grid.Col>
  <Grid.Col offset={2} span={2} style={{ paddingTop: '6px' }}>
    <h2 style={{ color: 'gray', marginTop: '0px', marginBottom: '8px' }}>{`${resourceType} IDs`}</h2>
  </Grid.Col>
</Grid>
<Divider my="md" style={{ marginTop: '14px' }} />
```

Reply:
Did not notice that at all- good catch!